pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -206,6 +297,17 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
         action: () => subvolume_create(volume, subvol, (mounted && !opt_ro) ? mount_point : mount_point_in_parent),
     });
 
+    let delete_excuse = create_excuse;
+    if (subvol.id === 5)
+        delete_excuse = _("Deleting the root subvolume is not allowed");
```

Comment:
We might consider not having the "delete" action for root at all. There is nothing the user can do to enable this action.

Reply:
Agreed.